### PR TITLE
'cat' here document rather than 'echo'

### LINKED ISSUE #2352: build scripts don't give a nice message when run with go < 1.4
```
hack/build-go.sh 
make: *** [all] Error 2
```

They should tell you 1.4 is required nicely

### PR DESCRIPTION
**Pull request by mjisyang**
Fixes #2352

### CODE REVIEW

File: hack/common.sh [hunk 1]
```diff
@@ -193,7 +193,7 @@ os::build::setup_env() {
   os::build::create_gopath_tree
 
   if [[ -z "$(which go)" ]]; then
-    echo <<EOF
+    cat <<EOF
```
Comment (liggitt) [inline hack/common.sh]: can you also change this on line 212? that would fix #2352

**Comment by mjisyang** [comment]
I think this is ready to merge, right?

**Comment by liggitt** [comment]
[merge]

**Comment by liggitt** [comment]
[merge]

**Pull request merged**